src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  #hidden = false;
+
+  hide() {
+    this.#hidden = true;
+  }
+
+  get hidden() {
+    return this.#hidden;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(animal) {
+    const isHerbivore = animal instanceof Herbivore;
+    const isHidden = animal.hidden;
```

Comment:
Accessing the private property `#hidden` directly from outside the `Herbivore` class is not allowed. Consider using the `hidden` getter method to check if the herbivore is hidden.